gnovm/pkg/gnolang/op_types.go

```diff
@@ -142,7 +142,7 @@ func (m *Machine) doOpInterfaceType() {
 	}
 	// push interface type
 	it := &InterfaceType{
-		PkgPath: m.Package.PkgPath,
+		PkgPath: m.Package.ModFile.Path,
```

Comment:
We should not implement mod and package versioning at the VM level. The module is a development tool that manages package versioning outside of VM execution.

Same as many other places in this implementation